test/TestProjects/MgmtRenameRules/readme.md

```diff
@@ -46,6 +45,6 @@ rename-mapping:
   SshPublicKeyResource: SshPublicKey
   LogAnalyticsOperationResult: LogAnalytics
   RollingUpgradeStatusInfo: VirtualMachineScaleSetRollingUpgrade
-  UpgradeOperationHistoricalStatusInfo.type: ResourceType
+  UpgradeOperationHistoricalStatusInfo.type: ResourceType|resource-type
```

Comment:
If I want to add a format for a property, but not rename it. Do I have to put `Type|resource-type`? Do you think it adds value if we support `|resource-type` to do format only?

Reply:
I explained this scenario in the document